tutorial-server/notebooks/CactusTutorial.ipynb

```diff
@@ -738,12 +763,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# This cell enables inline plotting in the notebook\n",
-    "%matplotlib inline\n",
```

Comment:
Why are we getting rid of %matplotlib inline?

Reply:
We already have `%matplitlib inline` in another cell, and this achieves no effects.